packages/template-retail-react-app/config/default.js

```diff
@@ -35,7 +35,8 @@ module.exports = {
             // This differs from the siteId in commerceAPIConfig for testing purposes
             siteId: 'aaij-MobileFirst',
             isProduction: false
-        }
+        },
+        preservePlusSignAsSpace: false
```

Comment:
"Preserve" means to keep the same. Your variable says something like keep this the same by making it different. You might want to think of something better. 

Also, since we are adding a configuration value, we might want to think about where this option lives. Right now you have it on "App.preservePlusSignAsSpace" ... as an outside reader that doesn't tell me as much information as I would like. Maybe we can name space it better. Example would be _app.urls.optionName_

Reply:
good point. I've changed it to `interpretPlusSignAsSpace` and moved it under app.url